content/docs/wrp/simple-messages.md

```diff
@@ -47,9 +54,10 @@ the same message type.
 Name | Description
 -----|--------------
 msg_type | The message type for the request-response.  (**SHALL** be 3.)
-source | The device_id name of the device originating the request or response.
-dest | The device_id name of the target device of the request or response.
-content_type | (optional) The media type of the payload.
+source | The device_id name of the device originating the request or response (i.e. dns:tr1d1um.example.net for inbound messages).
+dest | The device_id name of the target device of the request or response (i.e. mac:112233445566/config for inbound messages).
+device_id | The canonical device ID (i.e. mac:112233445566) involved in this envelope. It defers from source or dest in that no further parsing is required to extract the ID from the value.
+content_type | (optional) The media type of the payload. If not specified on inbound messages, `application/octet-stream` is used as default.
```

Comment:
I thought we were only adding this for events? Is this also being added to request-responses?

Reply:
Nevermind, found the pull request (https://github.com/xmidt-org/webpa-common/pull/508).  But isn't it filled in for outbound messages, not inbound?